compose.yaml

```diff
@@ -9,14 +14,16 @@ services:
       watch:
         - path: ./app
           action: rebuild
+    command: ["web", "--dev"]
+
   otel-collector:
     image: otel/opentelemetry-collector-contrib
     command: --config=/etc/otelcol-config.yaml
     ports:
-      - 8888:8888 # Prometheus metrics exposed by the Collector
-      - 8889:8889 # Prometheus exporter metrics
-      - 4317:4317 # OTLP gRPC receiver
-      - 4318:4318 # OTLP http receiver
+      - 8888:8888  # Prometheus metrics exposed by the Collector
+      - 8889:8889  # Prometheus exporter metrics
+      - 4317:4317  # OTLP gRPC receiver
+      - 4318:4318  # OTLP http receiver
```

Comment:
My yaml linter complains when there's only one space before an inline comment. That's all this is about. I can change it back if it's annoying.